Specifications/Events/for_EventStore/when_committing_a_public_object.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.SDK.Events.for_EventConverter.given;
+using Machine.Specifications;
+using It = Machine.Specifications.It;
+
+namespace Dolittle.SDK.Events.for_EventStore
+{
+    public class when_committing_a_public_object : given.an_event_store_and_an_execution_context
+    {
+        static an_event content;
+        static EventSourceId event_source;
+        static EventType event_type;
+
+        Establish context = () =>
+        {
+            content = new an_event("goodbye public world", 12345, true);
+            event_source = new EventSourceId(Guid.NewGuid());
+            event_type = new EventType(Guid.NewGuid());
+
+            event_types.Setup(_ => _.GetFor(content.GetType())).Returns(event_type);
```

Comment:
It seems like there should also be a test for when the `event_types` doesn't know about the type?